org.adempiere.base/src/org/compiere/print/DataEngine.java

```diff
@@ -1371,6 +1405,19 @@ else if (token.equals("LINE"))
 
 		return outStr.toString();
 	}
+	
+	/**
+	 * Returns Timestamp in format MM/DD/YYYY
+	 * @param Timestamp t
+	 * @return
+	 */
+	private String formatTimestamp(Timestamp t) {
+		if(t != null) {
+			String sTimestamp = t.toString();
+			return sTimestamp = sTimestamp.substring(5, 7) + "/" + sTimestamp.substring(8, 10) + "/" + sTimestamp.substring(0, 4);
```

Comment:
Wrong to hardcode the formatting here, please use DisplayType.getDateFormat for date or timestamp format instead.